great_expectations/cli/v012/util.py

```diff
@@ -2,47 +2,35 @@
 import re
 import sys
 from types import ModuleType
-from typing import Optional
+from typing import List, Optional
 
+import click
 import pkg_resources
 from pkg_resources import Distribution, WorkingSet
+from typing_extensions import Final
 
 from great_expectations.cli.v012 import toolkit
 from great_expectations.cli.v012.python_subprocess import (
     execute_shell_command_with_progress_polling,
 )
 from great_expectations.util import import_library_module, is_library_loadable
 
-try:
-    from termcolor import colored
-except ImportError:
-    colored = None
+SUPPORTED_CLI_COLORS: Final[List[str]] = ["blue", "cyan", "green", "yellow", "red"]
 
 
-def cli_message(string) -> None:
+def cli_message(string: str) -> None:
     print(cli_colorize_string(string))
```

Comment:
This entire file is duplicating efforts in the primary CLI directory - while this feels bad (and is bad), this code is outdated and will soon be removed from the codebase.

I don't think it is worth the effort to refactor existing code to refactor out this duplication. See a Slack conversation around this for more information: https://greatexpectationslabs.slack.com/archives/C9GDA5SAY/p1668435895377679